packages/wrangler/src/__tests__/cloudchamber/create.test.ts

```diff
@@ -62,6 +116,25 @@ describe("cloudchamber create", () => {
 		);
 	});
 
+	it("should fail with a nice message when parameters are mistyped", async () => {
```

Comment:
this is 🚀 . Config validation related tests should go in [configuration.test.ts](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/configuration.test.ts). IMO, we can create a dedicated `describe` block for Cloudchamber, like [d1](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/configuration.test.ts#L2194), [hyperdrive](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/configuration.test.ts#L2289) & co do

Reply:
in fact, leave this as is. I went ahead and added all necessary tests in this [commit](https://github.com/cloudflare/workers-sdk/pull/6300/commits/e3649bcfa8161768cc0c698cbcddda0f6d4c45bc) to make things easier ♥️ . Can you please have a look and let me know if that looks alright to you 🙏 ?